graphql_compiler/tests/test_macro_schema.py

```diff
@@ -46,25 +46,25 @@ def test_get_schema_for_macro_definition_addition(self) -> None:
         original_schema = self.macro_registry.schema_without_macros
         macro_definition_schema = get_schema_for_macro_definition(original_schema)
         macro_schema_directive_names = {
-            directive.name for directive in macro_definition_schema.get_directives()
+            directive.name for directive in macro_definition_schema.to_kwargs()["directives"]
```

Comment:
Let's try to avoid all of these `to_kwargs()` calls if possible and use direct attribute access.